scripts/components/HUD/Buttons/Button/Button.scss

```diff
@@ -121,7 +121,7 @@
       transform: translateX(130%) translateY(-50%) rotate(45deg);
     }
   }
-  .btn-wrap:hover .tooltip {
+  .btn-wrap:hover .tooltip, .btn-wrap:focus-within .tooltip {
     bottom: 140%;
   }
```

Comment:
Same as above

```suggestion
  .btn-wrap:hover .tooltip {
    bottom: 140%;
  }

  .btn-wrap:focus-within .tooltip {
    bottom: 140%;
  }
```

Reply:
Good catch! I tested this and I'll be commiting these changes right away!